client/blocks/login/index.jsx

```diff
@@ -488,20 +488,6 @@ class Login extends Component {
 							isBlazePro={ isBlazePro }
 							isWoo={ isWCCOM }
 						/>
-						<LoginForm
-							disableAutoFocus={ disableAutoFocus }
-							onSuccess={ this.handleValidLogin }
-							socialService={ socialService }
-							socialServiceResponse={ socialServiceResponse }
-							domain={ domain }
-							locale={ locale }
-							userEmail={ userEmail }
-							handleUsernameChange={ handleUsernameChange }
-							signupUrl={ signupUrl }
-							showSocialLoginFormOnly
```

Comment:
I think we can remove the prop `showSocialLoginFormOnly` and any associated logic completely now?

Reply:
Oh yes, there are no references left to it. Removed in: [`e23795a` (#105240)](https://github.com/Automattic/wp-calypso/pull/105240/commits/e23795a5813d7635cb537675cce1a0367729109d)